Laravel.gitignore

```diff
@@ -1,5 +1,7 @@
 vendor/
 node_modules/
+.rocketeer/
```

Comment:
These changes seem unrelated to the PR - can you tell me a bit about why they're useful here?